RecoLocalCalo/HGCalRecProducers/interface/HGCalCLUEAlgo.h

```diff
@@ -97,6 +97,8 @@ class HGCalCLUEAlgo : public HGCalClusteringAlgoBase {
     edm::ParameterSetDescription descNestedNoiseMIP;
     descNestedNoiseMIP.add<bool>("scaleByDose", false);
     iDesc.add<edm::ParameterSetDescription>("scaleByDose", descNestedNoiseMIP);
+    descNestedNoiseMIP.add<unsigned int>("scaleByDoseAlgo", 0);
+    iDesc.add<edm::ParameterSetDescription>("scaleByDoseAlgo", descNestedNoiseMIP);
     descNestedNoiseMIP.add<std::string>("doseMap", "");
     iDesc.add<edm::ParameterSetDescription>("doseMap", descNestedNoiseMIP);
```

Comment:
we already have the following in the config 
```python
    scaleByDose = cms.PSet(
      scaleByDose = cms.bool(False)
    ),
    doseMap = cms.PSet(
      scaleByDose = cms.bool(False),
      doseMap = cms.string('')
    ),
    noiseMip = cms.PSet(
      scaleByDose = cms.bool(False),
      doseMap = cms.string(''),
      noise_MIP = cms.double(0.01)
    ),
```
There will now be also ```scaleByDoseAlgo = cms.PSet(scaleByDose = False, scaleByDoseAlgo = 0)``` with ```scaleByDoseAlgo = 0``` added to all the following PSets.
is this nested accumulating content really desired?

Reply:
thanks @slava77 
I see there are 3 params added to the main pset which are not needed.
I've added a commit which fixes that - at least according to my understanding.
The relvals run.
@deguio (who committed the 3 lines I am removing) @rovere may comment, do you share the logic ?